Files/UserControls/Widgets/LibraryCards.xaml

```diff
@@ -22,20 +21,66 @@
         <StaticResource x:Key="ButtonBorderBrushDisabled" ResourceKey="ControlStrokeColorDefaultBrush" />
     </UserControl.Resources>
     <StackPanel Spacing="12">
-        <TextBlock
-            x:Uid="LibraryWidgetDescription"
-            HorizontalAlignment="Left"
-            FontFamily="Segoe UI"
-            FontSize="14"
-            FontWeight="Medium"
-            Opacity=".8"
-            Text="Folders" />
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition />
+                <ColumnDefinition />
+            </Grid.ColumnDefinitions>
+
+            <TextBlock
+                x:Uid="LibraryWidgetDescription"
+                HorizontalAlignment="Left"
+                FontFamily="Segoe UI"
+                FontSize="14"
+                FontWeight="Medium"
+                Opacity=".8"
+                Text="Folders"
+                TextTrimming="CharacterEllipsis" />
+
+            <!-- Options button -->
+            <Button
+                x:Uid="LibraryCardsOptionsButton"
+                Grid.Column="1"
+                Width="36"
+                Height="32"
+                Padding="0"
+                HorizontalAlignment="Right"
+                VerticalAlignment="Center"
+                HorizontalContentAlignment="Center"
+                VerticalContentAlignment="Center"
+                AutomationProperties.Name="More library options"
```

Comment:
Can you add the strings for this to the en-US resource file?

Reply:
I didn't forget it, I want to add them when I'm finished (see *update language files* at the end of the check list)

Reply:
@nvi9 Sounds good, great work!

Reply:
I've added (hopefully) all missing strings